pencil2d.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.1
+Type=Application
+Name=Pencil2D
+GenericName=Animation Software
+Comment=Create traditional hand-drawn animation using both bitmap and vector graphics
+Icon=pencil2d
+Exec=Pencil2D
```

Comment:
Probably, I added the desktop file mainly because linuxdeployqt wants one, so I didn’t really pay attention to that. I’ll go over it again and see how I can improve it for normal installation.